support-frontend/app/views/contributions.scala.html

```diff
@@ -55,8 +55,16 @@
       };
     }
 
-    window.guardian.supporterPlusProductPrices = @Html(outputJson(productPrices.supporterPlusProductPrices))
-    window.guardian.tierThreeProductPrices = @Html(outputJson(productPrices.tierThreeProductPrices))
+    /** Both the landing page (`/uk/contribution`) and contributions checkout (`/uk/contribution/checkout`) are rendered from this template.
+     *
+     * We use `supporterPlusProductPrices` and `tierThreeProductPrices` on the landing page as it renders both products with their promotions.
+     *
+     * We use `productPrices` in the checkout as the value is coupled across the codebase via redux state used across multiple checkouts i.e. Guardian Weekly, Paper, Contribution etc
```

Comment:
And this is only the T2 checkout right? Because T3 is using GW checkout currently, and generic checkout has something different?

Reply:
Yea, T2 @ `/uk/contribute/checkout` which uses `SupporterPlus` `productPrices`.

T3 @ `/subscribe/weekly/checkout` uses `productPrices` but from the [`GuardianWeekly` product](https://github.com/guardian/support-frontend/blob/main/support-frontend/app/controllers/WeeklySubscriptionFormController.scala#L68).